custom/icds_reports/migrations/0138_index_intermediate_tables.py

```diff
@@ -0,0 +1,176 @@
+                migrations.AlterIndexTogether(
+                    name='aggregatechildhealthpostnatalcareforms',
+                    index_together=set([('state_id', 'month')]),
+                ),
+            ]
+        ),
+        migrations.RunSQL(
+            sql=CREATE_INDEX_SQL.format(
+                index='icds_dashboard_child_hea_state_id_month_3050cc30_idx',
+                table=const.AGG_CHILD_HEALTH_THR_TABLE,
+                columns=','.join(('state_id', 'month'))
+            ),
+            reverse_sql=DROP_INDEX_SQL.format(index=''),
+            state_operations=[
+                migrations.AlterIndexTogether(
+                    name='aggregatechildhealththrforms',
+                    index_together=set([('state_id', 'month')]),
+                ),
+            ]
+        ),
+        migrations.RunSQL(
+            sql=CREATE_INDEX_SQL.format(
+                index='icds_dashboard_comp_feed_form_state_id_month_5cf21eec_idx',
+                table=const.AGG_COMP_FEEDING_TABLE,
+                columns=','.join(('state_id', 'month'))
+            ),
+            reverse_sql=DROP_INDEX_SQL.format(index=''),
+            state_operations=[
+                migrations.AlterIndexTogether(
+                    name='aggregatecomplementaryfeedingforms',
+                    index_together=set([('state_id', 'month')]),
+                ),
+            ]
+        ),
+        migrations.RunSQL(
+            sql=DROP_INDEX_SQL.format(index='icds_dashboard_growth_monitoring_forms_month_59c07619'),
+            reverse_sql=CREATE_INDEX_SQL.format(
+                index='icds_dashboard_growth_monitoring_forms_month_59c07619',
+                table=const.AGG_GROWTH_MONITORING_TABLE,
+                columns=['month']
+            ),
+            state_operations=[
+                migrations.AlterField(
+                    model_name='aggregategrowthmonitoringforms',
+                    name='month',
+                    field=models.DateField(help_text='Will always be YYYY-MM-01'),
+                ),
+            ]
+        ),
+        migrations.RunSQL(
+            sql=CREATE_INDEX_SQL.format(
+                index='icds_dashboard_growth_monitoring_forms_month_59c07619',
+                table=const.AGG_GROWTH_MONITORING_TABLE,
+                columns=','.join(('state_id', 'month'))
+            ),
+            reverse_sql=DROP_INDEX_SQL.format(index='icds_dashboard_growth_monitoring_forms_month_59c07619'),
+            state_operations=[
+                migrations.AlterIndexTogether(
+                    name='aggregategrowthmonitoringforms',
+                    index_together=set([('state_id', 'month')]),
+                ),
+            ]
+        ),
+        migrations.RunSQL(
+            sql=CREATE_INDEX_SQL.format(
+                index='icds_dashboard_daily_feeding_forms_state_id_month_273d19dd_idx',
+                table=const.AGG_DAILY_FEEDING_TABLE,
+                columns=','.join(('state_id', 'month'))
```

Comment:
This isn't super useful now, but it will be once I revert the daily feeding form agg to be state based once again.